### PR TITLE
Ledger consistant

### PR DESCRIPTION
**Pull request by xiemylogos**
modify ledger consistant base on consensus in vbft, sync block and submit block

### CODE REVIEW

File: consensus/vbft/service.go [hunk 1]
```diff
@@ -119,14 +120,17 @@ type Server struct {
...
-	sub        *events.ActorSubscriber
-	quitC      chan struct{}
-	quit       bool
-	quitWg     sync.WaitGroup
+	msgRecvC        map[uint32]chan *p2pMsgPayload
+	msgC            chan ConsensusMsg
+	bftActionC      chan *BftAction
+	msgSendC        chan *SendMsgEvent
+	sub             *events.ActorSubscriber
+	execResult      *store.ExecuteResult
```
Comment (Honglei-Cong) [inline consensus/vbft/service.go]: 可以把ExecuteResult保存到chain store中吗？

File: consensus/vbft/types.go [hunk 2]
```diff
@@ -56,6 +57,10 @@ func (blk *Block) getNewChainConfig() *vconfig.ChainConfig {
 	return blk.Info.NewChainConfig
 }
 
+func (blk *Block) getexecResMarkleRoot() common.Uint256 {
```
Comment (Honglei-Cong) [inline consensus/vbft/types.go]: Merkle ?

File: consensus/vbft/service.go (hunk 1 above)
Comment (xiemylogos) [reply consensus/vbft/service.go]: ok

File: consensus/vbft/types.go (hunk 2 above)
Comment (xiemylogos) [reply consensus/vbft/types.go]: yes

File: cmd/import_cmd.go [hunk 3]
```diff
@@ -157,7 +159,7 @@ func importBlocks(ctx *cli.Context) error {
 		if err != nil {
 			return fmt.Errorf("block height:%d deserialize error:%s", i, err)
 		}
-		err = ledger.DefLedger.AddBlock(block)
+		err = ledger.DefLedger.AddBlock(block, common.Uint256{})
```
Comment (laizy) [inline cmd/import_cmd.go]: need use `ExecuteBlock` and `SubmitBlock` instead

File: consensus/vbft/chain_store.go [hunk 4]
```diff
@@ -109,17 +129,18 @@ func (self *ChainStore) AddBlock(block *Block) error {
 //
 // SetBlock is used when recovering from fork-chain
 //
+/*
 func (self *ChainStore) SetBlock(block *Block, blockHash common.Uint256) error {
 
-	err := self.db.AddBlock(block.Block)
+	err := self.db.AddBlock(block.Block, common.Uint256{})
```
Comment (laizy) [inline consensus/vbft/chain_store.go]: remove commented code

**Pull request merged**